temporal-spring-boot-autoconfigure-alpha/src/main/java/io/temporal/spring/boot/autoconfigure/RootNamespaceAutoConfiguration.java

```diff
@@ -113,6 +114,11 @@ public WorkflowClient client(ClientTemplate clientTemplate) {
     return clientTemplate.getWorkflowClient();
   }
 
+  @Bean(name = "temporalWorkflowScheduleClient")
```

Comment:
```suggestion
  @Bean(name = "temporalScheduleClient")
```

I think it's clearer to just call it a schedule client